charts/admission-controller/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: admission-controller
 description: Sysdig Admission Controller using Sysdig Secure inline image scanner
 type: application
-version: 0.9.0
+version: 0.9.1
```

I think since this change is introducing a new component that is defaulted to enabled, the HPA, we should make this a minor release instead of a patch. This will require you to also do the version bumps in `sysdig-deploy/Chart.yaml` though.
```suggestion
version: 0.10.0
```

---

bumped minor version as it was suggested.